cmd/wksctl/profile/enable/enable.go

```diff
@@ -78,6 +82,23 @@ func profileEnableRun(params profileEnableFlags) error {
 	}
 	log.Info("Added the profile to the local repository.")
 
+	ignoreFilePath := path.Join(clonePath, constants.WKSctlIgnoreFilename)
+	if _, err := os.Stat(ignoreFilePath); err == nil {
+		log.Info("Ignoring files declared in .wksctlignore...")
```

Comment:
This log print should use the `WKSctlIgnoreFileName` constant instead of hardcoding the filename.

Reply:
ack

Reply:
fixed